src/components/OptionsSelector/index.android.js

```diff
@@ -0,0 +1,18 @@
+import React, {forwardRef} from 'react';
+import BaseOptionsSelector from './BaseOptionsSelector';
+import {propTypes, defaultProps} from './optionsSelectorPropTypes';
+
+const OptionsSelector = forwardRef((props, ref) => (
+    <BaseOptionsSelector
+        // eslint-disable-next-line react/jsx-props-no-spreading
+        {...props}
+        ref={ref}
+        shouldDelayFocus
+    />
+));
+
+OptionsSelector.propTypes = propTypes;
```

Comment:
I am getting this propType warning:

<img width="1123" alt="image" src="https://user-images.githubusercontent.com/1228807/187882944-afa7d64e-72ee-4347-98e4-3a7d6b5627f5.png">

I think it's because this component isn't using `withLocalize`. Can you please fix it?